front/lib/api/assistant/onboarding.ts

```diff
@@ -86,40 +86,97 @@ const ROLE_TO_PRIMARY_TOOL: Partial<
   // "other" intentionally omitted - will fall back to email-only or Notion default
 };
 
+const FAVORITE_PLATFORM_TO_TOOL_SID: Record<string, string> = {
+  google: "gmail",
+  microsoft: "outlook",
+  slack: "slack",
+  notion: "notion",
+  confluence: "confluence",
+  github: "github",
+  hubspot: "hubspot",
+  jira: "jira",
+  front: "front",
+};
+
+const TOOL_DISPLAY_NAMES: Record<string, string> = {
```

Comment:
We could kill that and use `asDisplayName` utils to get the display tool name 🙏🏻